src/main/java/io/javalin/Javalin.java

```diff
@@ -51,8 +51,8 @@
 
     private static Logger log = LoggerFactory.getLogger(Javalin.class);
 
-    private Server jettyServer = JettyServerUtil.defaultServer();
-    private SessionHandler jettySessionHandler = JettyServerUtil.defaultSessionHandler();
+    private Server jettyServer = null;
+    private SessionHandler jettySessionHandler = null;
```

Comment:
Add comments explaining this?

Reply:
The `defaultServer()` function will fail if Jetty is not on the classpath. The same with default session handler. Therefore I'm deferring the call to those only after somebody attempts to `javalin.start()`

Reply:
I know, but that's not obvious for new contributors reading the code. If we choose this approach we need to explain what's going on.

Reply:
You're right. Yet, Passing `jettyServer` and `jettySessionHandler` as constructor parameters seem to work pretty nicely, so I was able to remove the now-useless null check.